lab-Evan/index.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+const greet = require('./lib/greet.js');
+greet.greet('Evan');
+
+
+// COMMAND LINE UTILITY - EXTRA CREDIT
+const clArgument = process.argv[2];
+console.log(`Hey there from the ${clArgument}`);
```

Comment:
Since process.argv[2] is generated dynamically, it would be more appropriate to use the var key word. Const implies that a variable will not be reassigned, thus it will remain constant throughout the program.